strings.js

```diff
@@ -0,0 +1,5 @@
+var firstName = "Peter";
```

Comment:
hey king! I see that you are using var. i would like you to change all var to  let or const, that would be great!